src/indra_cogex/sources/pubmed/__init__.py

```diff
@@ -119,67 +213,30 @@ def _yield_journal_nodes(self) -> Iterable[Node]:
                     "abbr_title": journal_abbrev,
+                    # Should match issn_l in JournalPublisherProcessor
                     "issn_l": issn_l,
                     "p_issn": p_issn,
                     "e_issn": e_issn,
-                    "alternate_issn:string[]": ";".join(other),
+                    # Rely on data from JournalPublisherProcessor for the issn_list
+                    # "issn_list:string[]": ";".join(other),
                 }
                 yield Node(
                     "NLM",
                     nlm_id,
-                    labels=[self.journal_node_type],
+                    labels=[JOURNAL_NODE_TYPE],
                     data=data,
                 )
 
-    def get_relations(self) -> Iterable[List[Relation]]:
-        # Ensure cached files exist
-        # Todo: Add force option to download files?
-        process_mesh_xml_to_csv(
-            mesh_pmid_fpath=self.mesh_pmid_path,
-            pmid_year_types_fpath=self.pmid_year_types_path,
-            pmid_nlm_fpath=self.pmid_nlm_path,
-            journal_info_fpath=self.journal_info_path,
-        )
-        logger.info("Generating mesh-pmid relations")
-        yield from self._yield_mesh_pmid_relations()
-        logger.info("Generating pmid-journal relations")
-        yield from self._yield_pmid_journal_relations()
-
-    def _yield_mesh_pmid_relations(self) -> Iterable[List[Relation]]:
-        with gzip.open(self.mesh_pmid_path, "rt") as fh:
-            reader = csv.reader(fh)
-            next(reader)  # skip header
-            # NOTE tested with 100000 batch size but given that total is ~290M
-            # and each line is lightweight, trying with larger batch here
-            batch_size = 10_000_000
-            for batch in tqdm(
-                batch_iter(reader, batch_size=batch_size, return_func=list)
-            ):
-                relations_batch = []
-                for mesh_id, major_topic, pmid in batch:
-                    relations_batch.append(
-                        Relation(
-                            "PUBMED",
-                            pmid,
-                            "MESH",
-                            mesh_id,
-                            "annotated_with",
-                            data={
-                                "is_major_topic:boolean": get_bool(major_topic == "1")
-                            },
-                        )
-                    )
-                yield relations_batch
-
-    def _yield_pmid_journal_relations(self) -> Iterable[List[Relation]]:
+    def _get_relations(self) -> Iterable[List[Relation]]:
         # Yield batches of relations
         with gzip.open(self.pmid_nlm_path, "rt") as fh:
             reader = csv.reader(fh)
             # Skip header
             next(reader)
-            # The file has more than 35000000 lines - a batch size of 1M is
+            # The file has more than 3500000 lines - a batch size of 1M is
```

Comment:
This could be a typo?

Reply:
Resolved in 39bb0662540c859c1216207d3803c6a1c6627616